### ISSUE TITLE
WARNING: UNPROTECTED PRIVATE KEY FILE!

### ISSUE THREAD

**Issue by ChristianAdamski** [docksal/docksal#507]
Not found in issue queue:

When using drush sql-sync, I constanly get this warning

> @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
> @         WARNING: UNPROTECTED PRIVATE KEY FILE!          @
> @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
> Permissions 0644 for '/home/docker/.ssh/id_rsa' are too open.
> It is recommended that your private key files are NOT accessible by others.
> This private key will be ignored.
> key_load_private_type: bad permissions
> 

ALso reported (and reported fixed) for Lando: https://github.com/lando/lando/issues/268

**Comment by achekulaev** [comment]
Hi @ChristianAdamski. Please provide output of `fin config`.

**Comment by ChristianAdamski** [comment]
cap@ecomm-1003-vm:~/Dev/zuhause/zuhause$ fin config
WARNING: No swap limit support
---------------------
COMPOSE_PROJECT_NAME_SAFE: zuhause
COMPOSE_FILE:
/home/cap/.docksal/stacks/volumes-bind.yml
/home/cap/.docksal/stacks/stack-default.yml
/home/cap/Dev/zuhause/.docksal/docksal.yml
/home/cap/Dev/zuhause/.docksal/docksal-local.yml
ENV_FILE:
/home/cap/Dev/zuhause/.docksal/docksal.env
/home/cap/Dev/zuhause/.docksal/docksal-local.env

PROJECT_ROOT: /home/cap/Dev/zuhause
DOCROOT: zuhause
VIRTUAL_HOST: zuhause.docksal
VIRTUAL_HOST_ALIASES: *.zuhause.docksal
IP: 192.168.64.100
MYSQL:

Docker Compose configuration
---------------------
services:
  cli:
    dns:
    - 192.168.64.100
    - 10.11.1.1
    environment:
      DOCROOT: zuhause
      DRUSH_ALLOW_XDEBUG: '1'
      HOST_GID: '1000'
      HOST_UID: '1000'
      PHP_IDE_CONFIG: serverName=zuhause.docksal
      SECRET_ACAPI_EMAIL: null
      SECRET_ACAPI_KEY: null
      SECRET_SSH_PRIVATE_KEY: null
      SECRET_TERMINUS_TOKEN: null
      VIRTUAL_HOST: zuhause.docksal
      XDEBUG_CONFIG: remote_connect_back=0 remote_host=192.168.64.1
      XDEBUG_ENABLED: '1'
    hostname: cli
    image: docksal/cli:2.1-php7.1
    volumes:
    - docksal_ssh_agent:/.ssh-agent:ro
    - /home/docker
    - project_root:/var/www:rw,nocopy
  db:
    dns:
    - 192.168.64.100
    - 10.11.1.1
    environment:
      MYSQL_DATABASE: default
      MYSQL_PASSWORD: user
      MYSQL_ROOT_PASSWORD: root
      MYSQL_USER: user
    hostname: db
    image: ilakic/mariadb-docksal:mariadb-10.0
    ports:
    - 32771:3306/tcp
    volumes:
    - project_root:/var/www:ro,nocopy
  web:
    depends_on:
      cli:
        condition: service_started
    dns:
    - 192.168.64.100
    - 10.11.1.1
    environment:
      APACHE_BASIC_AUTH_PASS: null
      APACHE_BASIC_AUTH_USER: null
      APACHE_DOCUMENTROOT: /var/www/zuhause
    hostname: web
    image: docksal/web:2.1-apache2.4
    labels:
      io.docksal.project-root: /home/cap/Dev/zuhause
      io.docksal.virtual-host: zuhause.docksal,*.zuhause.docksal
    volumes:
    - project_root:/var/www:ro,nocopy
version: '2.1'
volumes:
  docksal_ssh_agent:
    external: true
    name: docksal_ssh_agent
  project_root:
    driver: local
    driver_opts:
      device: /home/cap/Dev/zuhause
      o: bind
      type: none

**Comment by achekulaev** [comment]
PR https://github.com/docksal/service-cli/pull/45

**Comment by achekulaev** [comment]
@lmakarov when it's clear please comment in which version of `cli` it is fixed.

**Comment by lmakarov** [comment]
This has been released in docksal/cli [v2.2.0](https://github.com/docksal/service-cli/releases/tag/v2.20) images:

```
docksal/cli:2.2-php5.6
docksal/cli:2.2-php7.0
docksal/cli:2.2-php7.1
docksal/cli:2.2-php7.2
```
